examples/cube.py

```diff
@@ -453,7 +453,7 @@ async def draw_frame_async():
 uniform_data = np.zeros((), dtype=uniform_dtype)
 
 
-if __name__ == "__main__":
+if True or __name__ == "__main__":
```

Comment:
Its probably been a long day for me working on graphcis, but I can't understand why this won't run....